packages/gafl-webapp-service/src/pages/contact/contact/route.js

```diff
@@ -69,13 +69,13 @@ const getContent = (permission, messages, junior) => {
   return isSalmonLicense ? messages.important_info_contact_content_salmon : messages.important_info_contact_content_not_salmon
 }
 
-const getMobileText = (permission, messages) =>
-  permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.text && permission.licenceLength === '12M'
+const getMobileText = (permission, messages, junior) =>
+  permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.text && permission.licenceLength === '12M' && !junior
     ? `${messages.important_info_contact_item_txt_value}${permission.licensee.mobilePhone}`
     : messages.important_info_contact_item_txt
 
-const getEmailText = (permission, messages) =>
-  permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.email && permission.licenceLength === '12M'
+const getEmailText = (permission, messages, junior) =>
+  permission.licensee.preferredMethodOfConfirmation === HOW_CONTACTED.email && permission.licenceLength === '12M' && !junior
```

Comment:
I think there might be benefit to having a one line `isTwelveMonthNonJuniorLicence` function rather than repeating the same (fairly complex) code three times